spec/models/user_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe User do
+
+    describe "needs_shave_or_trimming" do
+        it "should be false when status is present & shaved_at date is less than 3 days old" do
+            @user = User.create(first_name: "test", last_name: "user", email: Random.email, :gender => "male", :status => ["shaved", "trimmed", "massaged_and_trimmed", "unshaved"].sample, :shaved_at => DateTime.now)
+            needs_shave_or_trimming = @user.needs_shave_or_trimming
+            needs_shave_or_trimming.should be_falsy
+        end
+    end
+
+    describe "male?" do
+        it "should return true when user is male" do
+            @user = User.create(first_name: "test", last_name: "user", email: Random.email, :gender => "female", :status => ["shaved", "trimmed", "massaged_and_trimmed", "unshaved"].sample, :shaved_at => DateTime.now)
```

Comment:
`let` and `subject` are excellent helpers for keeping your test control variables well organised.